### PR TITLE
Add tasklist

### PR DESCRIPTION
**Pull request by oliverchubet**

### CODE REVIEW

**Review by ayjayt — DISMISSED** [review]
You obviously highly value project management

**Comment by oliverchubet** [comment]
> You obviously highly value project management

Does that mean it's too high-level? I can break it down.

File: todo/tasks.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+### Fill in missing values
+        
+* If it's a boolean value, you can make it equidistant from either category. 
+* If it's numerical, you can either put in an average/median value, discard the data point, or something else at your discretion.
+* If it's categorical, non-boolean, use your discretion dependent on the nature of the column.
+
+### Location data
+
+* Expand any location data to two columns for the coordinates
```
Comment (ayjayt) [inline todo/tasks.md:13]: Can I add in other more interesting correlations? Like is the State Red or Blue?

File: todo/tasks.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+### Rescale
+
+* The space should end up being an n-dimensional hypercube.
```
Comment (ayjayt) [inline todo/tasks.md:17]: lol obviously

**Comment by ayjayt** [comment]
@shpew12 It looks like the submodule was updated? Is that right. Doesn't seem right for this PR.

**Review by ayjayt — APPROVED** [review]

File: todo/tasks.md (hunk 1 above)
Comment (oliverchubet) [reply todo/tasks.md:13]: Yeah I guess so. Could be interesting.

**Pull request merged**